doc/ref_model/chapters/chapter10.md

```diff
@@ -40,8 +40,9 @@ Gaps
 
 This section addresses major open issues identified in the development of the Reference Model, Reference Architecture, Reference Implementation of the Common Cloud Infrastructure Lifecycle Framework. 
 
-## 10.3.1 Discovery [Initial language]
-The VNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. CNTT reference model specifies a model, but OpenStack does not provide APIs to support such capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
+## 10.3.1 Discovery
+The workloads (VNFs/CNFs) and Cloud Infrastructure should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is the VNF/CNF onboarding - both in terms of time and complexity. It could take weeks or months to onboard a VNF/CNF. There are lots of static and laborious checks performed to ensure the compatibility of the workloads with the corresponding Cloud Infrastructure. 
```

Comment:
Is the "weeks or months to onboard a VNF/CNF" due to an inability to discover "capabilities"? If not, then this doesn't belong here.

"checks performed to ensure the compatibility of the workloads with the corresponding Cloud Infrastructure" -- but in the case of "CNTT conformant workloads" many of these checks would already have been performed (onboarding success is the driver fro workload conformance).

for consistency:
```suggestion
The workloads (VNFs/CNFs) and Cloud Infrastructure should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is workload onboarding - both in terms of time and complexity. It can take weeks or months to onboard a network function workload (VNF/CNF). There are a number of static and laborious checks that need to be performed to ensure the compatibility of the workloads with the Cloud Infrastructure. 
```